index.html

```diff
@@ -34,8 +34,9 @@ <h1>Tests for Web Bluetooth</h1>
 
   <h2>Tests</h2>
   <ol>
-    <li><a href="characteristic_read">Simple read-only characteristic</a></li>
-    <li><a href="characteristic_readwrite">Simple read/write characteristic</a></li>
+    <li><a href="characteristic_read">Read characteristic</a></li>
+    <li><a href="characteristic_readwrite">Read and write to characteristic</a></li>
+    <li><a href="characteristic_notify">Subscribe to a characteristic changes</a></li>
```

Comment:
"Subscribe to characteristic value changes"

Reply:
Done.